discord/raw_models.py

```diff
@@ -97,9 +97,13 @@ class RawMessageDeleteEvent(_RawReprMixin):
         The message ID that got deleted.
     cached_message: Optional[:class:`Message`]
         The cached message, if found in the internal message cache.
+    data: :class:`dict`
+        The raw data given by the `gateway <https://discord.com/developers/docs/topics/gateway-events#message-delete>`_.
```

Comment:
I think a different verb like "sent" would fit better in these cases. What do you say?